src/support/spawnStreamAsync.ts

```diff
@@ -0,0 +1,254 @@
+import { spawn, SpawnOptions } from 'child_process';
+import * as os from 'os';
+//import * as treeKill from 'tree-kill';
+import { ShellQuotedString, ShellQuoting } from 'vscode';
+import { IShell } from './Shell';
+import { ChildProcessError } from './ChildProcessError';
+import { CommandLineArgs } from './commandLineBuilder';
+
```

Comment:
can we put a comment at the top of the file linking to the original source of this code from the docker extension? That way it'll be easier for future maintainers to know where the source is.

Reply:
This applies to all the files that were copied